backend/pkg/server/server.go

```diff
@@ -36,10 +36,14 @@ func New(
 	restaurantHandler := restaurantHandler{restaurantService:restaurantService}
 
 	r.Route("/api", func(r chi.Router) {
-		r.Route("/restaurant", func(r chi.Router) {
+		r.Route("/exists", func(r chi.Router) {
```

Comment:
a co tu się zadziało? :D

Reply:
ta metoda exists była napisana tylko po to, żeby pokazać architekturę, raczej nie chcemy jej wystawiać na świat :)